cmd/podman/main_urfave.go

```diff
@@ -0,0 +1,237 @@
+// +build urfave
```

Comment:
Should this be dropped from the final patch set?

Reply:
yup!